apps/api/prisma/schema.prisma

```diff
@@ -306,18 +306,63 @@ model Folder {
 }
 
 model Document {
-  guid                 String   @id @default(uuid())
+  guid                 String             @id @default(uuid())
   name                 String
-  folder               Folder   @relation(fields: [folderId], references: [id])
+  folder               Folder             @relation(fields: [folderId], references: [id])
   folderId             Int
   blobStorageContainer String?
   blobStoragePath      String?
-  status               String   @default("awaiting_upload")
-  redacted             Boolean  @default(false)
-  isDeleted            Boolean  @default(false)
-  createdAt            DateTime @default(now())
-  fileSize             Int      @default(0)
+  status               String             @default("awaiting_upload")
+  redacted             Boolean            @default(false)
+  isDeleted            Boolean            @default(false)
+  createdAt            DateTime           @default(now())
+  fileSize             Int                @default(0)
   fileType             String?
+  DocumentMetadata     DocumentMetadata[]
 
   @@unique([name, folderId])
 }
+
+model DocumentMetadata {
+  id Int @id @unique @default(autoincrement())
+
+  Document     Document? @relation(fields: [documentGuid], references: [guid])
+  documentGuid String
+
+  caseRef String?
+
+  horizonDataID String?
+
+  version String
+
+  path             String?
+  virusCheckStatus String?
+
+  fileMD5 String?
+  mime    String
+
+  fileSize Int     @default(0)
+  fileType String?
+
+  dateCreated  DateTime
+  lastModified DateTime?
+
+  datePublished DateTime?
+
+  documentType String
+
+  published Boolean @default(false)
+
+  securityClassification String?
+  sourceSystem           String
+  origin                 String
+
+  owner  String?
+  author String?
+
+  representative String
+  description    String
+  stage          Int?
```

Comment:
Will this refer to the stage of the case during upload? If so, I would say that this could be a little more descriptive. Otherwise it can be confused with Document Status

Reply:
I need to look more into it and find out that why this is a draft pr so we can all discuss this

Reply:
Stage is a value that is applied to the document when it is uploaded to the system.  It marks what stage in the case this document applied to.  It will not change as the case progresses.